libexec/swiftenv-install

```diff
@@ -81,7 +81,17 @@ install_tar_binary() {
   mkdir -p "$TMPDIR/swiftenv-$VERSION"
 
   echo "Downloading $URL"
-  curl "$URL" -s | tar xz -C "$TMPDIR/swiftenv-$VERSION"
+
+  if $verify; then
+    pushd "$TMPDIR/swiftenv-$VERSION"
+    curl -O "$URL"
+    curl -O "$URL.sig"
+    gpg --verify "swift-$VERSION"*.sig
+    tar xzf "swift-$VERSION"*.tar.gz
```

Comment:
It's just matching the downloaded sig and tar files, the actual file names aren't captured anywhere AFAIK so I base the match on convention. It could probably be simplified since only one of each would be downloaded into that temp dir, at least I think that's how it works.